lib/vmpooler/pool_manager.rb

```diff
@@ -50,7 +50,10 @@ def check_pending_vm(vm, pool, timeout)
             rescue
             end
 
+            finish = '%.2f' % (Time.now - Time.parse($redis.hget('vmpooler__vm__' + vm, 'clone')))
+
             $redis.smove('vmpooler__pending__' + pool, 'vmpooler__ready__' + pool, vm)
+            $redis.hset('vmpooler__boot__' + Date.today.to_s, pool + ':' + vm, finish)
```

Comment:
Should there be a separator between `Date.today.to_s` and `pool`? (for later string parsing/splitting/chopping)